cmd/clusterctl/config/zz_generated.bindata.go

```diff
@@ -95,7 +95,7 @@ func (fi bindataFileInfo) Sys() interface{} {
d5\xe4\xef\x5d\x4f\xa2\xd8\xc7\x0e\x38\x7b\xbf\x00\x60\xec\xa9\x83\x98\xc2\xde\x59\x4a\xd2\x18\x9f\x45\x29\x19\xf5\xe3\xcf\xe6\xf3\x72\x38\xd2\x42\x22\x99\x82\x6e\x9b\x42\x8e\x1d\x3c\xe5\x3a\x24\x1e\xcf\x82\x4a\xdb\x90\xdc\xb8\x5e\x8e\xa1\x4b\x8c\xae\x5a\x06\xa6\x6e\x8f\x28\xaa\xc9\x3b\xd1\x9f\x66\xe6\x9f\x9d\x68\xdd\x8a\x3e\x27\xf4\x13\xd4\xd5\x2a\x8e\xb7\xd9\x63\x3a\xdb\x17\x00\x62\x42\xa4\x0e\x7e\x2d\x60\x22\x1a\xb2\x0b\x80\x23\x79\x15\xcc\x12\xd0\xda\xda\x0e\xf4\xb7\xc9\xb1\x52\xba\x09\x3e\xf7\x7c\x82\xfa\x51\x02\xdf\xa2\xee\x3a\x68\xf4\x10\xa9\x5a\x47\xda\xde\x9f\x0d\x65\xaf\x03\xd1\xe4\x78\xfb\x38\x72\x44\x54\x70\xcc\x32\xcc\x8e\xfc\xb5\x2c\x47\xe0\xb3\x04\xf7\x33\xdb\xd3\xf1\x9f\x50\xcd\x8e\xec\x89\x8c\x59\xa2\x0f\x65\x13\x2e\xf7\x1e\x25\x1c\x9c\xf7\x57\xe8\xe3\x0e\x5f\x0d\xc4\x9b\x1d\xf5\xd8\x1d\x23\x42\x24\xfe\xfe\xf6\xed\xfd\xab\xbb\x99\x19\xc0\x92\x98\xe4\xa2\xd6\x7b\x3b\x9e\x1b\x6c\x99\x07\x12\x40\x06\x62\x4d\x07\x70\x0c\xba\xa3\x53\x0f\xc1\xf1\x9e\x58\x43\x3a\x34\xa7\x4c\x31\x85\x48\x49\x4f\xf7\x69\xf8\x26\x13\x39\xb1\x5e\xd4\x7d\x59\xa0\x0d\x5e\xa7\xd2\xa5\xdc\x91\x5a\xb2\xc7\xd3\x40\xd8\x80\xee\x9c\x40\xa2\x98\x48\x88\x87\xe1\x9c\x25\x86\xe2\x84\x0c\x61\xfd\x91\x8c\x36\x70\x47\xa9\xa4\x01\xd9\x85\xec\x6d\x99\xe0\x3d\x25\x85\x44\x26\x6c\xd9\xfd\x79\xca\x2d\xa0\xa1\x16\xf5\xa8\x74\xbc\xd2\xe7\xaf\xde\x41\x46\x0f\x7b\xf4\x99\xfe\x0f\xc8\x16\x7a\x3c\x40\xa2\x52\x05\x32\x4f\xf2\x55\x17\x69\xe0\x97\x90\x08\x1c\x6f\x42\x07\x3b\xd5\x28\xdd\x6a\xb5\x75\x3a\x2a\x91\x09\x7d\x9f\xd9\xe9\x61\x55\x45\xc5\xad\xb3\x86\x24\x2b\x4b\x7b\xf2\x2b\x71\xdb\x25\x26\xb3\x73\x4a\x46\x73\xa2\x15\x46\xb7\xac\xd0\xb9\xaa\x51\xd3\xdb\xff\xa5\xa3\x76\xc9\xcb\x19\xd6\x47\x77\x63\xf8\xea\x2c\x3f\xd1\x81\x32\xd4\xe0\x04\xf0\x18\x3a\x9c\xe2\x4c\x74\x31\x15\x76\xde\xfd\x78\xf7\x1e\xc6\xd2\xb5\x19\x97\xec\x57\xde\xcf\x81\x72\x6e\x41\x21\xcc\xf1\x86\xd2\xd0\xc4\x4d\x0a\x7d\xcd\x49\x6c\x63\x70\xac\x75\x61\xbc\x23\xbe\xa4\x5f\xf2\xba\x77\x5a\xfa\xfe\x47\x26\xd1\xd2\xab\x06\x6e\xaa\x3c\xc3\x9a\x20\x47\x8b\x4a\xb6\x81\xb7\x0c\x37\xd8\x93\xbf\x41\xa1\xff\xbc\x01\x85\x69\x59\x16\x62\x9f\xd7\x82\xe9\x3f\xcb\xa5\xf3\xc0\xda\x64\x63\xaa\x4d\x4f\xf4\xed\x76\xe2\x06\x8e\xad\x2b\xaa\x3e\x0c\x4f\xd1\x84\x61\x60\xce\x73\xdb\x3c\x07\x67\x35\xff\x7b\xc9\x22\xaf\x17\xa5\x4a\xc4\xa3\x52\x70\x47\x74\xc2\x37\xd1\xe4\xf1\xdb\x84\x04\x58\xff\x50\x4a\xa8\xe4\x18\x43\xd2\xd3\xf8\x3c\x07\xe8\xfe\xab\xc2\x32\x8a\xca\x1c\xae\x09\x7d\x0c\x4c\xac\x63\x86\x67\xd1\x72\xa9\xd3\x4f\x94\xfd\x70\xe1\xfa\x85\xce\x0c\xf6\x4f\x3b\x4a\x34\xa3\xed\x82\xa4\xf3\x83\xa3\x0c\xa7\x93\x01\x85\xe3\x6d\x03\x6e\x03\xd4\x47\x3d\xcc\x85\x79\x1e\x30\x7a\x7f\x81\xfa\xe1\xc6\x49\x51\x76\xf4\xfe\x8c\x49\x9e\x41\xc6\xa3\x3b\x2b\x65\xc2\x6d\x07\x9a\xf2\xd0\x66\xd1\x90\x70\x4b\x53\x4b\x5e\x9f\x14\xab\x83\xbf\xfe\x5e\x88\xa2\xe6\xfa\x5f\x81\xc6\x50\xd4\x23\x5f\xdd\xe4\xe9\xf1\xe2\xc5\xec\x65\x51\x97\x26\xf0\xf0\x34\x90\x0e\x7e\xfb\x7d\x31\x94\x22\x7b\x3f\x3e\x1f\x8a\xf1\x9f\x00\x00\x00\xff\xff\x0c\x4b\x3d\xcf\x07\x0a\x00\x00")
```

Comment:
these changes shouldn't be generated, see https://github.com/kubernetes-sigs/cluster-api/pull/4162#issuecomment-775824516

Reply:
Thanks for the comment, I've cleaned those. But some tests are still failing.